configs/example/xpk.Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Dockerfile for xpk examples in xpk_example_dag.py,
+# and is saved at gcr.io/cloud-ml-auto-solutions/xpk_jax_test:latest.
+FROM python:3.10
+RUN pip install --upgrade pip
+RUN pip install "jax[tpu]" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+RUN pip install --no-cache-dir --upgrade clu tensorflow tensorflow-datasets
+RUN git clone https://github.com/google/flax.git /tmp/flax
+RUN pip install --no-cache-dir flax
+RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
+RUN mkdir -p /usr/local/gcloud \
+  && tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz \
+  && /usr/local/gcloud/google-cloud-sdk/install.sh
+ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
+RUN gcloud components install kubectl
```

Comment:
Does the test image itself need to have `kubectl` installed?

Reply:
Good question. The example image is used in 2 places:

1) image for [KubernetesPodOperator](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/implementations/utils/xpk.py#L79),
2) image to trigger E2E model test by [xpk](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/implementations/utils/xpk.py#L69).

I think this is needed by the 1st case. But I will double check.

Reply:
Yes, it's needed for the operator; otherwise, we will get the error below.

Will also make the comment there.

```
b'WARNING: Accessing a Kubernetes Engine cluster requires the kubernetes commandline\nclient [kubectl]. To install, run\n  $ gcloud components install kubectl\n\nFetching cluster endpoint and auth data.\nCRITICAL: ACTION REQUIRED: gke-gcloud-auth-plugin, which is needed for continued use of kubectl, was not found or is not executable. Install gke-gcloud-auth-plugin for use with kubectl by following https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke\nkubeconfig entry generated for mas-v4-8.\n/bin/sh: 1: kubectl: not found\n'
```